### ISSUE TITLE
Feature request : Face Mask detection system using Computer Vision 

### ISSUE THREAD

**Issue by 17arindam** [UppuluriKalyani/ML-Nexus#160]
**Is your feature request related to a problem? Please describe.**
A Face mask detection model which will identify whether someone has a mask or not 

**Describe the solution you'd like**
Face Mask Detection System built with OpenCV, Keras/TensorFlow using Deep Learning and Computer Vision concepts in order to detect face masks in static images as well as in real-time video streams.

**Additional context**
![image](https://github.com/user-attachments/assets/80fddf18-624e-4a3a-be25-9533cfdcb6d4)

**Comment by 17arindam** [comment]
@UppuluriKalyani can you assign me this issue
